examples/examples_load_optimization/03_row_opt_example.py

```diff
@@ -0,0 +1,149 @@
+"""Example: Optimize a row of turbines
+
+This example optimizes the derating of a row of three turbines to maximize net revenue for a
+variety of combinations of wind direction, load ambient TI, and electricity values.
+The row is aligned when the wind direction is 270 degrees.
+
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from floris import FlorisModel, TimeSeries
+from floris.optimization.load_optimization.load_optimization import (
+    compute_farm_revenue,
+    compute_farm_voc,
+    compute_net_revenue,
+    find_A_to_satisfy_rev_voc_ratio,
+    optimize_power_setpoints,
+)
+
+
+# Parameters
+D = 126.0
+d_spacing = 7.0
+MIN_POWER_SETPOINT = 0.00000001
+derating_levels = np.linspace(1.0, MIN_POWER_SETPOINT, 10)
+n_turbines = 3
+N_per_loop = 10 # Number of unique values for wind direction, value, and load ambient TI
+
+
+# Declare a floris model with default configuration
+fmodel = FlorisModel(configuration="defaults")
+fmodel.set_operation_model("simple-derating")
+
+
+# Set up a row of turbines
+fmodel.set(
+    layout_x=[i * D * d_spacing for i in range(n_turbines)],
+    layout_y=[0.0 for i in range(n_turbines)],
+)
+
+
+# Set up input conditions
+wind_directions = []
+values = []
+load_ambient_tis = []
+
+for w_i in np.linspace(230, 270, N_per_loop):
+    for v_i in np.linspace(1e-5, 5e-5, N_per_loop):
+        for t_i in np.linspace(0.02, 0.2, N_per_loop):
+            wind_directions.append(w_i)
+            values.append(v_i)
+            load_ambient_tis.append(t_i)
+wind_directions = np.array(wind_directions)
+values = np.array(values)
+load_ambient_tis = np.array(load_ambient_tis)
+N = len(wind_directions)
+
+time_series = TimeSeries(
+    wind_directions=wind_directions, wind_speeds=8.0, turbulence_intensities=0.06, values=values
+)
+
+# Run the FLORIS model
+fmodel.set(wind_data=time_series)
+fmodel.run()
+
+# Set the initial power setpoints as no derating
+initial_power_setpoint = np.ones((N, n_turbines)) * 5e6
```

Comment:
Would this be a good place to demonstrate the `get_max_powers` function from the load_optimization module?